observability/logging/logging-6.0/log6x-otel-data-model.adoc

```diff
@@ -0,0 +1,374 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-otel-data-model"]
+= OTEL data model
+include::_attributes/common-attributes.adoc[]
+:context: log6x-otel-data-model
+
+toc::[]
+
+This document outlines the protocol and semantic conventions for Red Hat OpenShift Logging's OTEL support with Logging v6.1, which is in Technology Preview.
+
+:FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
+include::snippets/technology-preview.adoc[]
+
+== Forwarding and Ingestion Protocol
+
+Red Hat OpenShift Logging collects and forwards logs to OpenTelemetry endpoints using link:https://opentelemetry.io/docs/specs/otlp/[OTLP]. OTLP encodes, transports, and delivers telemetry data. This product can also deploy Loki storage, which provides an OTLP endpont to ingest log streams. This document defines the semantic conventions for the logs collected from various OpenShift cluster sources.
+
+[NOTE]
+====
+Logs are forwarded using OTLP over HTTP, as defined by the OpenTelemetry Observability Framework, with Protobuf payloads encoded in JSON format.
+====
+
+== Semantic Conventions
+
+The log collector in this solution gathers the following log streams:
+
+* Container logs
+* Cluster node journal logs
+* Cluster node auditd logs
+* Kubernetes and OpenShift API server logs
+* OpenShift Virtual Network (OVN) logs
+
+These streams are forwarded according to the sementatic conventions defined by OpenTelemetry semantic attributes. The semantic conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry, identified by *Attributes*. For example, a process running in a container includes attributes such as container_name, cluster_id, pod_name, namespace, and possibly deployment or app_name. These *Attributes* are grouped in the *Resource* object, which helps reduce repetition and optimizes log transmission as telemetry data.
```

Comment:
It seems a bit weird that we explain *Resource attributes* but we skip *Scope attributes* and *Log attributes* seems like this would be relevant information for the users. I would suggest expanding this section slightly to cover the other two.